src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * An interactive ChatBot that provides functions of a to-do list
+ * depending on the user's input.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Contructor to create an instance of Duke.
+     *
+     * @param filePath the String representation of the relative file path to store the data
```

Comment:
@param filePath String representation of the relative file path to store the data